manifests/c/ChenQingYu/BF/1.0.0/ChenQingYu.BF.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ChenQingYu.BF
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Chen QingYu
+PublisherUrl: https://github.com/chen-qingyu
+PublisherSupportUrl: https://github.com/chen-qingyu/bf/issues
+Author: Chen QingYu
+PackageName: Brainfuck interpreter
+PackageUrl: https://github.com/chen-qingyu/bf
+License: GPL-3.0
+LicenseUrl: https://github.com/chen-qingyu/bf/blob/main/license
+ShortDescription: A simple Brainfuck interpreter implemented in C.

```

```suggestion
ShortDescription: A simple Brainf**k interpreter implemented in C.
```
This appears to be a "radio-safe" common spelling for the language. Is this change acceptable?

---

Of course, thank you.